I can't build it on Fedora | SDL2_MAIN_LIBRARY-NOTFOUND

Hello, 

I've got every SDL2 package that I could find installed on my Fedora 31 but in the end of the `make` command it complains like this:

```
make[2]: *** No rule to make target 'SDL2_MAIN_LIBRARY-NOTFOUND', needed by 'Source/woof'.  Stop.
make[1]: *** [CMakeFiles/Makefile2:207: Source/CMakeFiles/woof.dir/all] Error 2
make: *** [Makefile:152: all] Error 2
```

Any idea what package I'm missing?

Thank you. 

EDIT: this is the list of packages the I've got installed:

```
dnf list installed *sdl*
Installed Packages
SDL.x86_64                                                     1.2.15-42.fc31                                        @updates-testing
SDL-devel.x86_64                                               1.2.15-42.fc31                                        @fedora         
SDL2.i686                                                      2.0.12-1.fc31                                         @updates        
SDL2.x86_64                                                    2.0.12-1.fc31                                         @updates        
SDL2-devel.x86_64                                              2.0.12-1.fc31                                         @updates        
SDL2_image.x86_64                                              2.0.5-2.fc31                                          @fedora         
SDL2_image-devel.x86_64                                        2.0.5-2.fc31                                          @fedora         
SDL2_mixer.x86_64                                              2.0.4-4.fc31                                          @fedora         
SDL2_mixer-devel.x86_64                                        2.0.4-4.fc31                                          @fedora         
SDL2_net.x86_64                                                2.0.1-9.fc31                                          @fedora         
SDL2_net-devel.x86_64                                          2.0.1-9.fc31                                          @fedora         
SDL2_ttf.x86_64                                                2.0.15-2.fc30                                         @fedora         
SDL_image.x86_64                                               1.2.12-22.fc31                                        @fedora         
SDL_image-devel.x86_64                                         1.2.12-22.fc31                                        @fedora         
SDL_mixer.x86_64                                               1.2.12-18.fc31                                        @fedora         
SDL_mixer-devel.x86_64                                         1.2.12-18.fc31                                        @fedora         
mingw64-SDL2.noarch                                            2.0.12-1.fc31                                         @updates 
```

--- a/Source/CMakeLists.txt
+++ b/Source/CMakeLists.txt
@@ -89,6 +89,11 @@ if(m_FOUND)
     list(APPEND WOOF_LIBRARIES m)
 endif()
 
+# Some platforms have SDL2main, others don't.
+if(SDL2_MAIN_LIBRARY)
+    list(APPEND WOOF_LIBRARIES SDL2::SDL2main)
+endif()
+
 if(WIN32)
     # Stamp out and compile resource file on Windows.
     configure_file(resource.rc.in "${CMAKE_CURRENT_BINARY_DIR}/resource.rc")
@@ -100,7 +105,7 @@ add_executable(woof WIN32 ${WOOF_SOURCES})
 target_woof_settings(woof)
 target_include_directories(woof PRIVATE "${CMAKE_CURRENT_BINARY_DIR}/../")
 target_link_libraries(woof PRIVATE ${WOOF_LIBRARIES}
-    ZLIB::ZLIB SDL2::SDL2main SDL2::SDL2 SDL2::image SDL2::mixer SDL2::net)
+    ZLIB::ZLIB SDL2::SDL2 SDL2::image SDL2::mixer SDL2::net)
 
 if(MSVC)
     # MSVC tries to supply a default manifest and complains when it finds ours

